tests/multi_ptr/multi_ptr_comparison_op.h

```diff
@@ -0,0 +1,657 @@
+            value_mptr < nullptr;
+      };
+
+      run_test(queue, run_test_action, expected_results);
+    }
+    SECTION(section_name(
+                "Check multi_ptr operator>(std::nullptr_t, const multi_ptr&)")
+                .with("T", type_name)
+                .with("address_space", address_space_name)
+                .with("decorated", is_decorated_name)
+                .create()) {
+      // Variable that contains all variables that will be used to verify test
+      // result
+      detail::mptr_nullptr_mptr_test_results<T> expected_results;
+      // We expect that "nullptr > nullptr_mptr" result should be equal to
+      // std::greater<multi_ptr_t>()(nullptr, nullptr_mptr) result
+      expected_results.nullptr_nullptr_mptr_result_val = true;
+      // We expect that "nullptr_mptr > nullptr" result should be equal to
+      // std::greater<multi_ptr_t>()(nullptr_mptr, nullptr) result
+      expected_results.nullptr_mptr_nullptr_result_val = true;
+      // We expect that "nullptr > value_mptr" result should be equal to
+      // std::greater<multi_ptr_t>()(nullptr, value_mptr) result
+      expected_results.nullptr_value_mptr_result_val = true;
+      // We expect that "value_mptr > nullptr" result should be equal to
+      // std::greater<multi_ptr_t>()(value_mptr, nullptr) result
+      expected_results.value_mptr_nullptr_result_val = true;
+
+      const auto run_test_action = [](auto arr_acc, auto result_acc) {
+        multi_ptr_t nullptr_mptr(nullptr);
+        multi_ptr_t value_mptr(arr_acc);
+
+        auto &test_result = result_acc[0];
+
+        test_result.nullptr_nullptr_mptr_result_val =
+            std::greater<multi_ptr_t>()(nullptr, nullptr_mptr) ==
+            nullptr > nullptr_mptr;
+        test_result.nullptr_mptr_nullptr_result_val =
+            std::greater<multi_ptr_t>()(nullptr_mptr, nullptr) ==
+            nullptr_mptr > nullptr;
+        test_result.nullptr_value_mptr_result_val =
+            std::greater<multi_ptr_t>()(nullptr, value_mptr) ==
+            nullptr > value_mptr;
+        test_result.value_mptr_nullptr_result_val =
+            std::greater<multi_ptr_t>()(value_mptr, nullptr) ==
+            value_mptr > nullptr;
+      };
+
+      run_test(queue, run_test_action, expected_results);
+    }
+    SECTION(section_name(
+                "Check multi_ptr operator<=(std::nullptr_t, const multi_ptr&)")
+                .with("T", type_name)
+                .with("address_space", address_space_name)
+                .with("decorated", is_decorated_name)
+                .create()) {
+      // Variable that contains all variables that will be used to verify test
+      // result
+      detail::mptr_nullptr_mptr_test_results<T> expected_results;
+      // We expect that "nullptr <= nullptr_mptr" result should be equal to
+      // std::less_equal<multi_ptr_t>()(nullptr, nullptr_mptr) result
+      expected_results.nullptr_nullptr_mptr_result_val = true;
```

Comment:
Can we remove these lines with expected true results and add some verify function that just checks that all results are `true` in these cases?

Reply:
I've done this in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/363/commits/0e236ae91a5b4d8ae52bbec9e668fac614d48216